modules/ossm-operator-install.adoc

```diff
@@ -0,0 +1,85 @@
+
+Starting with {ProductVersion} {ProductVersion}, you must install the Jaeger Operator and the Kiali Operator before the {ProductName} Operator can install the control plane.
+
+
+[id="ossm-operator-install-jaeger_{context}"]
+== Installing the Jaeger Operator
+You must install the Jaeger Operator for the {ProductName} Operator to install the control plane.
+
+.Prerequisites
+* An account with cluster administrator access.
+
+.Procedure
+
+. Log into your {product-title} installation as a cluster administrator.
+
+. Run the following commands to install the Jaeger Operator:
++
+----
+$ oc new-project observability # create the project for the jaeger operator
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/crds/jaegertracing_v1_jaeger_crd.yaml
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/service_account.yaml
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/role.yaml
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/role_binding.yaml
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/operator.yaml
+----
+
+[id="ossm-operator-install-kiali_{context}"]
+== Installing the Kiali Operator
+You must install the Kiali Operator for the {ProductName} Operator to install the control plane.
+
+.Prerequisites
+
+* Review the link:https://www.kiali.io/documentation/getting-started[Kiali] documentation.
+
+* An account with cluster administrator access.
+
+.Procedure
+
+. Log into your {product-title} installation as a cluster administrator.
+
+. Run the following command to install the Kiali Operator:
++
+----
+$ bash <(curl -L https://git.io/getLatestKialiOperator) --operator-image-version v1.0.0 --operator-watch-namespace '**' --accessible-namespaces '**' --operator-install-kiali false
+----
+
+
+[id="ossm-operator-install-istio_{context}"]
+== Installing the {ProductName} Operator
+
+.Prerequisites
+
+* Review the link:https://github.com/Maistra/istio-operator/tree/maistra-0.12.0/deploy[Operator templates on GitHub].
+
+* An account with cluster administrator access.
+
+* The Jaeger Operator must be installed.
+
+* The Kiali Operator must be installed.
+
+.Procedure
+
+. Log into your {product-title} installation as a cluster administrator.
+
+. If the `istio-operator` and `istio-system` namespaces do not exist, run these commands to create the namespaces:
++
+----
+$ oc new-project istio-operator
+$ oc new-project istio-system
+----
+
+. Run this command to install the {ProductShortName} Operator. You can run them from any host with access to the cluster.
++
+----
+$ oc apply -n istio-operator -f ./deploy/maistra-operator.yaml
```

Comment:
should this be a permanent link like the Jaeger and Kiali ones? Right now you have to check out the repo to make this work, which is not explicitly mentioned

Reply:
@brian-avery What is the best way to handle these links for TP12?

Reply:
Fixed.